github/git_refs.go

```diff
@@ -45,7 +47,8 @@ type updateRefRequest struct {
 	Force *bool   `json:"force"`
 }
 
-// GetRef fetches the Reference object for a given Git ref.
+// GetRef fetches a single Reference object for a given Git ref.
+// If there is no exact match, GetRef will return an error.
 //
 // GitHub API docs: https://developer.github.com/v3/git/refs/#get-a-reference
```

Comment:
> > Do you think maybe we should make the error say something about using GetRefs if you want to handle 0...N refs?
>
> No need. The documentation says that.

Well, I guess I lied about that. The docs doesn't really say that. So I agree it'd be an improvement to add a paragraph here, something that mentions that users should consider `GetRefs` if they want a method that is a more direct mapping to the GitHub API endpoint, if they want to be able to get multiple refs.